tutorials/asr/Lhotse_Dataloading.ipynb

```diff
@@ -0,0 +1,459 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c14eb898-0f8e-4783-a70e-ab54b4de910d",
+   "metadata": {},
+   "source": [
+    "# Tutorial: NeMo & Lhotse Data Loading\n",
+    "\n",
+    "![image](https://raw.githubusercontent.com/lhotse-speech/lhotse/master/docs/logo.png)\n",
+    "\n",
+    "In this tutorial we introduce the integration of NeMo with Lhotse, a library for speech data preparation and loading. Lhotse adds new capabilities to NeMo, allowing to move certain operations such as bucketing or dataset blending to dataloading runtime, rather than dataset preparation. This allows a greater flexibility in trying out various dataset setups without the need to prepare different data variants ahead of time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "57eada3f-d3df-4038-b927-c666ec2b8ab9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pickle\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import nemo\n",
+    "import lhotse\n",
+    "from lhotse import CutSet\n",
+    "from lhotse.recipes import (\n",
+    "    download_librispeech, \n",
+    "    download_yesno, \n",
+    "    prepare_librispeech, \n",
+    "    prepare_yesno,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "afd2b7ad-be34-4518-aa74-44e8253c0a1d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nemo_root = str(Path(nemo.__path__[0]).parent)\n",
+    "root_dir = Path(\"data\")\n",
+    "root_dir.mkdir(parents=True, exist_ok=True)\n",
+    "num_jobs = os.cpu_count() - 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6709798-78ec-477e-86da-8110f307f6f5",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "### Quick Lhotse Primer\n",
+    "\n",
+    "[Lhotse](https://github.com/lhotse-speech/lhotse) is a toolkit for speech data handling that originated as a part of the next-generation Kaldi framework. Next-gen Kaldi includes other tools, such as [k2](https://github.com/k2-fsa/k2), which is also [integrated into NeMo](https://github.com/NVIDIA/NeMo/blob/main/examples/asr/experimental/k2/speech_to_text_bpe.py).\n",
+    "\n",
+    "Lhotse represents speech data as python objects, and provides efficient sampling and dataloading mechanisms. The core Lhotse concepts leveraged in NeMo integration are:\n",
+    "- Representation for individual examples: [Recording](https://lhotse.readthedocs.io/en/latest/corpus.html#recording-manifest), [SupervisionSegment](https://lhotse.readthedocs.io/en/latest/corpus.html#supervision-manifest), and [Cut](https://lhotse.readthedocs.io/en/latest/cuts.html#cuts).\n",
+    "- Representation for a dataset and/or mini-batch: [CutSet](https://lhotse.readthedocs.io/en/latest/api.html#lhotse.cut.CutSet).\n",
```

Comment:
I think maybe "Representations of individual examples", "Representations of datasets and/or mini-batches"